pepdbagent/exceptions.py

```diff
@@ -36,3 +36,21 @@ def __init__(self, msg=""):
 class FilterError(PEPDatabaseAgentError):
     def __init__(self, msg=""):
         super().__init__(f"""pepdbagent filter error. {msg}""")
+
+
+class ProjectNotInFavorites(PEPDatabaseAgentError):
+    """
+    Project doesn't exist in favorites
+    """
+
+    def __init__(self, msg=""):
+        super().__init__(f"""Project is not in favorites list. {msg}""")
+
+
+class ProjectAlreadyInFavorites(PEPDatabaseAgentError):
```

Comment:
Should we raise an `Exception` for this case? I feel like if a user has a project starred, and for whatever reason they decide they want to star it again, we should just either 1) do nothing, or 2) overwrite the action, no?